extensions/src/main/java/org/pentaho/platform/plugin/services/security/userrole/ChainedUserDetailsService.java

```diff
@@ -51,7 +52,7 @@ public ChainedUserDetailsService( List<UserDetailsService> delegates ) {
         if ( details != null ) {
           return details;
         }
-      } catch ( UsernameNotFoundException ignored ) {
+      } catch ( UsernameNotFoundException | CommunicationException ignored ) {
         // ignore and continue;
```

Comment:
While we're fixing this, may as well add a log.debug() statement to at least be able to debug when this happens.